07-location.Rmd

```diff
@@ -42,11 +42,11 @@ Typical research questions include:
 This chapter demonstrates how geocomputation can answer such questions based on a hypothetical case study with based on real data.
 This case study is described in the next section.
 
-## Case study
+## Case study: bike shops in Germany
 
-Imagine your are launching a cycling company and would like to open shops across urban areas of Germany.
-Imagine further that a survey you conducted indicates following target group: males between 20-40 years old who live alone or with just one person (single households, not families).
-Additionally, you are in the lucky position to have sufficient capital to open a number of shops.
+Imagine you are starting a chain of bike shops in Germany.
+A survey suggests that single males between aged 20 to 40 are most likely to buy your products: this is the *target audience*.
+You are in the lucky position to have sufficient capital to open a number of shops.
```

Comment:
How many? I think it's important to provide a ball-park figure so the reader has a sense of the size of the challenge.